### ISSUE TITLE
Can I increase the training batch size?

### ISSUE THREAD

**Issue by aaaaaaapp** [nnnth/UniLIP#15]
“Image Generation and Editing Training. Stages 1, 2, and 3 are trained for 50k, 200k, and 30k steps, respectively. All stages use a batch size of 512 with a cosine learning rate decay schedule ranging from 1e-4 to 1e-5.“
Doubling the batch size on a single GPU—how will that affect the accuracy of the training results? Have you tried this? And what about using more nodes?
Thank for your great work!

**Comment by nnnth** [comment]
I haven't experimented with a larger batch size yet. However, a larger batch size generally means the model is less likely to be misled by specific bad samples (outliers), which could lead to better performance. Similar suggestions in [blip3o](https://github.com/JiuhaiChen/BLIP3o/issues/27#issuecomment-2982638392).
